ui.tests/test-module/specs/emailinput/emailinput.authoring.spec.js

```diff
@@ -0,0 +1,98 @@
+/*
+ *  Copyright 2022 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+const sitesSelectors = require('../../libs/commons/sitesSelectors'),
+    afConstants = require('../../libs/commons/formsConstants');
+
+/**
+ * Testing EmailInput with Sites Editor
+ */
+describe('Page - Authoring', function () {
+  // we can use these values to log in
+
+  const dropEmailInputInContainer = function() {
+    const dataPath = "/content/forms/af/core-components-it/blank/jcr:content/guideContainer/*",
+        responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-path='" + dataPath + "']";
+    cy.selectLayer("Edit");
+    cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Email input", afConstants.components.forms.resourceType.formemailinput);
+    cy.get('body').click( 0,0);
+  }
+
+  const dropEmailInputInSites = function() {
+    const dataPath = "/content/core-components-examples/library/adaptive-form/textinput/jcr:content/root/responsivegrid/demo/component/container/*",
```

Comment:
we alresdy have /content/core-components-examples/library/adaptive-form/emailinput in the examples in this PR, so why using textinput. we should keep collaterals separate for each component.